File: dayFive/javaScript/afterBreak/questionTwo.js
let array = [2,2,1,6,7];
let result = 0;
let count = 0
let countRepetition = 0
   
for(; count < array.length; count++){
    for(let counter = 0; counter < array.length; counter++){  
        if(array[count] == array[counter]){ 
            countRepetition+=1       
        }

     }

     if(countRepetition == 1){
        result = array[count];
        console.log(result);  
      } 
       
      countRepetition = 0
     
   }  
   
 
        

File: dayFive/python/afterBreak/question_two.py
array = [2,2,1,6,7]
result = 0;
countRepetition = 0
holder = 0
   
for count in range(len(array)):
    for counter in range(len(array)):  
        if(array[count] == array[counter]):
            holder = array[count]
            countRepetition+=1  
    if(countRepetition == 1):
    print(holder)


    countRepetition = 0
     
  
   
 
        
